spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/security/reactive/ReactiveManagementWebSecurityAutoConfigurationTests.java

```diff
@@ -117,6 +118,22 @@ public void backsOffIfCustomSecurityIsAdded() {
 				});
 	}
 
+	@Test
+	public void backsOffIfReactiveOAuth2ResourceServerAutoConfigurationSecurityIsAdded() {
+		this.contextRunner
+				.withConfiguration(AutoConfigurations
+						.of(ReactiveOAuth2ResourceServerAutoConfiguration.class))
+				.withPropertyValues(
+						"spring.security.oauth2.resourceserver.jwt.jwk-set-uri=http://authserver")
+				.run((context) -> {
+					assertThat(
+							getAuthenticateHeader(context, "/actuator/health").toString())
```

Comment:
I think asserting on the header is sort of relying on the behavior of `ReactiveOAuth2ResourceServerAutoConfiguration` which this test doesn't need to know about. I might be enough to just assert that the context doesn't have a bean for `ReactiveManagementWebSecurityAutoConfiguration`. WDYT?